Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -165,6 +165,14 @@ void EnhancementsWidget::LoadSettings()
       m_pp_effect->setCurrentIndex(m_pp_effect->count() - 1);
   }
 
+  const bool supports_postprocessing = g_Config.backend_info.bSupportsPostProcessing;
+  m_pp_effect->setEnabled(supports_postprocessing);
+
+  if (!supports_postprocessing)
```

Comment:
This if statement should be braced as it's more than one line long.

Reply:
Done.